features/external_istiod.md

```diff
@@ -135,7 +135,7 @@ No API added in 1.8
 - [x] Documentation on istio.io includes performance expectations; may have caveats. 
```

Comment:
Are the unchecked items here already completed?

Reply:
Will be, sort of :) I've add a discussion of this to the next TOC agenda.

Reply:
New integration tests: https://github.com/istio/istio/pull/34331